src/_utils/offline/service-worker/sw.template.js

```diff
@@ -0,0 +1,57 @@
+const cacheName = "<%= cacheName %>"
+
+/* eslint-disable quotes */
+const cacheList = JSON.parse('<%= files %>')
```

Comment:
From what I understand, for now, it's not different than appcache: all files (or only files from globby) are cached initially.
So service worker does not bring any value compared to appcache.

One thing we can do (I think it should be easy), is to accept 2 patterns: files that must be cached asap (current behavior) + files that can be cached when viewed.

Eg: putaindecode.io is a (big) website, and I don't want to cache all the files, all posts. But caching only viewed posts is a pretty nice feature, what do you think?